src/rest/transport/request.ts

```diff
@@ -13,14 +13,10 @@ export interface IPolygonEdgeHeaders extends Record<string, string> {
 
 export type IHeaders = IPolygonEdgeHeaders | Record<string, string>
 
-export interface IPolygonQueryWithCredentials extends IPolygonQuery {
-  apiKey: string | boolean;
-}
-
 export const auth =
-  (apiKey, func: Function, apiBase: string, headers?: IHeaders) =>
-  (...args) =>
```

Comment:
`...args` is putting all of the parameters passed into the function into an array, and the spread on the next line is turning them from an array back into multiple parameters being passed to the next function. I believe this change to stop spreading args would lead to only the first parameter of the function being passed along to the next function.

Consider this example:
`
const x = (...args) => console.log(args);
x(1,2,3)
// prints [1, 2, 3] because all parameters have been captured in an array

const y = (args) => console.log(args)
y(1,2,3)
// prints 1 because we've just named the first parameter being passed in "args"
`